schema/ddl/20240312173399_create_account.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS accounts (
+    id char(26) NOT NULL PRIMARY KEY,
+    total_amount INT NOT NULL,
+    store_id char(26) NOT NULL,
+    purchase_date TIMESTAMP NOT NULL,
```

Comment:
[imo] 購入日は過去なので，こうが良いと思います！👀
```suggestion
    purchased_at timestamptz NOT NULL,
```